src/Data/THGen/Orphans.hs

```diff
@@ -0,0 +1,9 @@
+module Data.THGen.Orphans
+where
+
+import qualified Data.Text as Text
+import           Data.Scientific
+import           Text.XML.Writer
+
+instance ToXML Scientific where
+  toXML = content . Text.pack . show
```

Comment:
`show` can use exponent if it feels like it. Using `formatScientific` might be better.